app/src/main/res/layout/layout_safe_item.xml

```diff
@@ -47,44 +61,15 @@
             android:layout_height="wrap_content"
             android:ellipsize="end"
             android:maxLines="2"
+            android:textColor="@color/gnosis_dark_blue_alpha_70"
```

Comment:
same here

Reply:
Should I create a style for this `textColor`?